Composer/packages/tools/language-servers/language-understanding/src/LUServer.ts

```diff
@@ -243,6 +245,38 @@ export class LUServer {
     this.LUDocuments.push(luDocument);
   }
 
+  protected async findAllImportedEntities(
+    imports: { id: string; path: string; description: string }[],
+    importResolver: (source: string, id: string) => Promise<{ id: string; content: any }>,
+    luFeatures: any
+  ): Promise<string[]> {
+    let content = '';
+    const result: string[] = [];
+    for (const importFile of imports) {
+      try {
+        content = (await importResolver('.', importFile.id)).content;
+      } catch (error) {
+        // ignore if file not exist
+      }
+
+      const { resource } = await this.luParser.parse(content, importFile.id, luFeatures);
```

Comment:
> await this.luParser.parse [](http://example.com/codeflow?start=26&length=26)

maybe it is safer to put this inside the try block as well?  According to [here](https://github.com/microsoft/botframework-cli/blob/7e5fa93174a86f61467d6fc3c867d0cf36d8d8c3/packages/lu/src/parser/lufile/luParser.js#L193)
This will be `undefined` if content is ''.  Then the destructuring will probably throw an error

Reply:
added try catch block